server/.eslintrc

```diff
@@ -6,7 +6,7 @@
     "tsconfigRootDir": "."
   },
   "plugins": ["@typescript-eslint/eslint-plugin"],
-  "extends": ["plugin:@typescript-eslint/recommended"],
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier"],
```

Comment:
I got this error when linting.
"ESLint couldn't find the config "prettier" to extend from"

Reply:
if you do npm install from server folder, that issue should be resolved